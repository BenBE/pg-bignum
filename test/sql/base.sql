\set ECHO none
BEGIN;
\i sql/bignum.sql
\set ECHO all

SELECT bignum('0'), bignum('1'), bignum('-1');

SELECT 0::bignum, 1::bignum, (-1)::bignum;

SELECT (1::int8)::bignum, (-1::int8)::bignum;

ROLLBACK;
