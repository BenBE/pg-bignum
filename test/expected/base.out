\set ECHO none
psql:sql/bignum.sql:14: NOTICE:  return type bignum is only a shell
psql:sql/bignum.sql:19: NOTICE:  argument type bignum is only a shell
SELECT bignum('0'), bignum('1'), bignum('-1');
 bignum | bignum | bignum 
--------+--------+--------
 0      | 1      | -1
(1 row)

SELECT 0::bignum, 1::bignum, (-1)::bignum;
 bignum | bignum | bignum 
--------+--------+--------
 0      | 1      | -1
(1 row)

SELECT (1::int8)::bignum, (-1::int8)::bignum;
 bignum | bignum 
--------+--------
 1      | -1
(1 row)

ROLLBACK;
